lib/services/authRestService.js

```diff
@@ -60,7 +60,7 @@ class AuthRestService {
 
   static create(ctx) {
 
-    return new AuthRestService(ctx, ctx.serversConfig);
+    return new AuthRestService(ctx, ctx.globalConfig);
```

Comment:
@richardcbrown `hostConfig` and `globalConfig` are different things. A single `hostConfig` is a part of `globalConfig` that contains information to connect a specific endpoint only.

Please pass `ctx.globalConfig.auth` in case of `AuthRestService` and `ctx.globalConfig.api` in case of `ResourceRestService`